nvtabular/ops/data_stats.py

```diff
@@ -56,8 +56,8 @@ def fit(self, col_selector: ColumnSelector, ddf: dd.DataFrame):
             self.col_types.append(col_type)
 
             # Get cardinality for cats
-            if col_type == "cats":
-                dask_stats[col]["cardinality"] = ddf[col].nunique()
+            if col_type != "conts":
+                dask_stats[col]["cardinality"] = ddf[col].nunique().compute()
```

Comment:
We shouldn't be calling '.compute' here (this is handled by the workflow, and the results are passed back to fit_finalize).

```suggestion
                dask_stats[col]["cardinality"] = ddf[col].nunique()
```